en/xml/fo/topic.graphic-titlepage.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    License CC BY-NC-SA 3.0
+
+   This work is licensed under the
+   "Namensnennung – Keine kommerzielle Nutzung – Weitergabe unter
+    gleichen Bedingungen 3.0 Deutschland (CC BY-NC-SA 3.0)"
+   http://creativecommons.org/licenses/by-nc-sa/3.0/de/deed.de
+
+   Read the English translation here:
+
+   "Attribution-NonCommercial-ShareAlike 3.0 Unported (CC BY-NC-SA 3.0)"
+   http://creativecommons.org/licenses/by-nc-sa/3.0/
+
+-->
+<?xml-model href="file:../5.1/dbref.rnc" type="application/relax-ng-compact-syntax"?>
+<section xml:id="dbc.fo.graphic-titlepage" remap="topic" userlevel="easy"
+    xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
+    xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Graphics on Title Pages</title>
+    <subtitle>Place and format an individual graphic on a title page</subtitle>
+    <info>
+        <definitions definitionfile="defs.xml"/>
+        <keywordset>
+            <keyword>title page</keyword>
+            <keyword>cover</keyword>
+            <keyword>graphic</keyword>
+            <keyword>block</keyword>
+            <keyword>format</keyword>
+        </keywordset>
+        <subjectset>
+            <subject>
+                <subjectterm>graphic</subjectterm>
+            </subject>
+        </subjectset>
+    </info>
+
+    <section role="problem">
+        <title>Problem</title>
+        <para>You need to know how to add a graphic to your titlepage.</para>
+    </section>
+    <section role="solution">
+        <title>Solution</title>
+        <para>To add a graphic to one of your titlepages, you need to creata a custom layer for
+            titlepages as described in the <xref linkend="dbc.fo.design-titlepages"/>section.</para>
+        <para>To add your graphic, the XSL-FO specification has defined an fo:block-container to
+            encapsulate an 'external-graphics' element. Additionally, there are many many options to
+            modify your graphic on the title pages like scaling or positioning.</para>
+        <programlisting language="xml">
+&lt;fo:block space-before="1em" space-after="1em">
```

Comment:
It's a minor detail, but you shouldn't add a linebreak between `<programlisting>` and your text. Remove the linebreak and start it like this:

```xml
<programlisting language="xml">&lt;fo:block space-before="1em" space-after="1em">
```